### PR TITLE
Removed attribute addpathfield as Model_Contact field type is not use…

### PR DESCRIPTION
**Pull request by deibit**
…d by the form

Pull Request for Issue #34198 .

### Summary of Changes

Removed attribute _addfieldpath_ from _fieldset_ tag as filter form does not use Modal_Contact field type.

### CODE REVIEW

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on 4d721b0e4f0e9fdc1b8c01cabd3f8a58d9e6360e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34203">issues.joomla.org/tracker/joomla-cms/34203</a>.</sub>

**Comment by joomdonation** [comment]
Testing instructions:

1. Use Joomla 3
2. Login to frontend of site using a super admin account
3. Edit an existing article
4. On edit screen, click on **Contact** from editor to insert a contact into article
5. On popup screen, try to use search tools to filter contacts and make sure it still works

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 4d721b0e4f0e9fdc1b8c01cabd3f8a58d9e6360e<br /><br />@joomdonation thanks for testing instructions.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34203">issues.joomla.org/tracker/joomla-cms/34203</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34203">issues.joomla.org/tracker/joomla-cms/34203</a>.</sub>

**Comment by HLeithner** [comment]
Why do we keep an empty fieldset ?

**Comment by brianteeman** [comment]
@HLeithner you're correct - it should be removed

**Comment by richard67** [comment]
Back to pending due to updates requested.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34203">issues.joomla.org/tracker/joomla-cms/34203</a>.</sub>

File: components/com_contact/models/forms/filter_contacts.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <form>
-<fieldset addfieldpath="/administrator/components/com_contacts/models/fields" />
+<fieldset/>
 	<fields name="filter">
```
Comment (Quy) [inline components/com_contact/models/forms/filter_contacts.xml]:
```suggestion
	<fields name="filter">
```

**Comment by Quy** [comment]
@zero-24 Can this still be RTC or retesting?

**Comment by zero-24** [comment]
There is no such thing as "com_contacts" just "com_contact" in the administrator path so it was never working in the first place ;) 

Will merge here thanks @deibit and all the others here for testing :)

**Pull request merged**